hashing/Cargo.toml

```diff
@@ -19,10 +19,13 @@ hex-buffer-serde = "0.3.0"
 itertools = "0.10.1"
 schemars = "0.8.3"
 serde = "1.0.130"
+thiserror = "1.0.29"
```

Comment:
Can you explain why we need to introduce a new crate just for errors?

Reply:
I don't think we need to, it's probably a matter of personal preference.

I inherited it from the following commit:
https://github.com/xcthulhu/casper-node/commit/c1f68d33daa569232d57c1c2f505a454d0025621#diff-5b8f4434a67c5c4d12119a080a9c0f80e5332cf2a25c9fda3be45cadc5f73ceb

@xcthulhu you think we can live without `thiserror` in `hashing`?

Reply:
As per discussion with the team, we're leaving `thiserror` here.